CODEOWNERS

```diff
@@ -0,0 +1 @@
+crates/lib/src/qpu/experimental @erichulburd
```

Comment:
I don't have push permissions to this repository, which is probably related to the error reported by Github here.